source/io/module_httpClient.js

```diff
@@ -44,8 +44,7 @@
 
         // WebVI Specific
         WEBVI_UNSUPPORTED_INPUT: 363798,
-        INVALID_HEADER: 363798,
-        FORBIDDEN_HEADER_WARNING: 363798
+        INVALID_HEADER: 363798
```

Comment:
I would leave the comma at the end even though this is the last item (as a convention). That way, when items are added after the last item or the last item is removed, the diff won't show an extra line just because of a comma. Many groups have this convention for many programming languages.